internal/experiment/webconnectivityqa/dnshijacking.go

```diff
@@ -27,7 +27,7 @@ func dnsHijackingToProxyWithHTTPURL() *TestCase {
 		},
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
-			DNSConsistency:  "inconsistent",
+			DNSConsistency:  "consistent",
```

Comment:
Here the DNS flips from inconsistent to consistent because we can perform a TLS handshake for the target domain to the spoofed IP address, so we say that the IP address is good for the domain.